skipper.go

```diff
@@ -318,6 +318,9 @@ type Options struct {
 	// DefaultFilters will be applied to all routes automatically.
 	DefaultFilters *eskip.DefaultFilters
 
+	// A list of filters unavailable for use
+	DisabledFilters []string
```

Comment:
Proper godoc starts with the name of the field. 
So: `DisabledFilters is a list of ...`